### PR TITLE
WT-6862 Remove duplicates in statistics descriptions

### PR DESCRIPTION
**Pull request by clarissecheah**
… and data source statistics

### CODE REVIEW

File: dist/stat_data.py [hunk 1]
```diff
@@ -668,8 +566,6 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_set_ts_stable_upd', 'set timestamp stable updates'),
     TxnStat('txn_sync', 'transaction sync calls'),
     TxnStat('txn_timestamp_oldest_active_read', 'transaction read timestamp of the oldest active reader', 'no_clear,no_scale'),
-    TxnStat('txn_update_conflict', 'update conflicts'),
-
```
Comment (tetsuo-cpp) [inline dist/stat_data.py:672]: Accidentally removed the newline before the new heading.

File: dist/stat_data.py [hunk 2]
```diff
@@ -737,59 +633,9 @@ def __init__(self, name, desc, flags=''):
...
-    CacheStat('cache_bytes_dirty', 'tracked dirty bytes in the cache', 'no_clear,no_scale,size'),
-    CacheStat('cache_bytes_dirty_total', 'bytes dirty in the cache cumulative', 'no_clear,no_scale,size'),
-    CacheStat('cache_bytes_inuse', 'bytes currently in the cache', 'no_clear,no_scale,size'),
-    CacheStat('cache_bytes_read', 'bytes read into cache', 'size'),
-    CacheStat('cache_bytes_write', 'bytes written from cache', 'size'),
-    CacheStat('cache_eviction_checkpoint', 'checkpoint blocked page eviction'),
-    CacheStat('cache_eviction_clean', 'unmodified pages evicted'),
-    CacheStat('cache_eviction_deepen', 'page split during eviction deepened the tree'),
-    CacheStat('cache_eviction_dirty', 'modified pages evicted'),
+
```
Comment (tetsuo-cpp) [inline dist/stat_data.py:636]: No need for this newline after the heading.

File: dist/stat_data.py [hunk 3]
```diff
@@ -897,13 +723,114 @@ def __init__(self, name, desc, flags=''):
...
+    RecStat('rec_suffix_compression', 'internal page key bytes discarded using suffix compression', 'size'),
+    RecStat('rec_time_window_pages_prepared', 'pages written including at least one prepare'),
+    RecStat('rec_time_window_pages_start_ts', 'pages written including at least one start timestamp'),
+    RecStat('rec_time_window_prepared', 'records written including a prepare'),
+
+    ##########################################
+    # Session operations
+    ##########################################
+    SessionOpStat('session_compact', 'object compaction'),
+
```
Comment (tetsuo-cpp) [inline dist/stat_data.py:737]: Remove newline.

File: dist/stat_data.py [hunk 4]
```diff
@@ -897,13 +723,114 @@ def __init__(self, name, desc, flags=''):
...
+    CacheStat('cache_inmem_splittable', 'in-memory page passed criteria to be split'),
+    CacheStat('cache_pages_requested', 'pages requested from the cache'),
+    CacheStat('cache_read', 'pages read into cache'),
+    CacheStat('cache_read_deleted', 'pages read into cache after truncate'),
+    CacheStat('cache_read_deleted_prepared', 'pages read into cache after truncate in prepare state'),
+    CacheStat('cache_read_overflow', 'overflow pages read into cache'),
+    CacheStat('cache_write', 'pages written from cache'),
+    CacheStat('cache_write_hs', 'page written requiring history store records'),
+    CacheStat('cache_write_restore', 'pages written requiring in-memory restoration'),
+  ##########################################
```
Comment (tetsuo-cpp) [inline dist/stat_data.py:800]: Weird indentation here.

**Review by tetsuo-cpp — COMMENTED** [review]
Awesome work @clarissecheah. Had some minor formatting comments.

**Review by tetsuo-cpp — APPROVED** [review]

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**